Makefile

```diff
@@ -335,24 +335,28 @@ artifact: build
 .PHONY: images
 images: spire-server-image spire-agent-image k8s-workload-registrar-image oidc-discovery-provider-image
 
+# The use of --ulimit memlock=-1 here is the workaround for
+# https://github.com/golang/go/issues/37436 when building on linux 5.3
+# and later
```

Comment:
This has since happened, so I've removed the --ulimit flag for now and rewritten this into a comment about the future necessity, for whoever next steps on this landmine.